src/alexjs/alex.handler.ts

```diff
@@ -4,12 +4,9 @@ import { Message } from 'discord.js';
 import { AlexService } from './alex.service';
```

Comment:
The File is called `alex.handler.ts`. It potentially can be named `alex.controller.ts` as its a controller for discord-events. The thing here is, that technically this file isnt a 'real' nest.js-Controller. In the Modules, theres the possibility to add a `controller`-property which would result in an error. I tried to prevent this errorby naming it `handler`. What do you think about that?

Reply:
I think `handler` is ok. We can always rename and tweak if we need to later on